concreteproperties/stress_strain_profile.py

```diff
@@ -688,6 +688,16 @@ def get_yield_strength(
 
         return self.yield_strength
 
+    def get_fracture_strain(
```

Comment:
Yes I think this works, provided `min(self.strains)` is always equal to the fracture strain. 

There is currently no built in steel stress/strain profiles where this is not the case. If it turns out to be the case in the future for any reason then can re-add `get_fracture_strain()` method to base class `StressStrainProfile` returning `self.fracture_strain`, and for concrete a similar one that returns `self.ultimate_strain`. 

That way you're returning the exact values specified in generating the stress/strain relationship which is what I was aiming for as I'm using those values to generate new modified properties suitable for different analysis types vs from the generated stress/strain relationship. There is a subtle difference potentially depending on future stress/strain profiles and whether or not `max(self.strains) == self.ultimate_strain` & `min(self.strains) == self.fracture_strain`.

Unless you think they should just be added now?

Reply:
Actually to avoid any future issues, I think I'll update the code to simply use `__getattribute__()` to grab the `stress_strain_profile` or `ultimate_stress_strain_profile` stored variables for `fracture_strain`, `ultimate_strain`, `yield_strength`, `compressive_strength`, etc... to get the exact values used to generate the materials and stress/strian profiles. 

I think this is a better approach as not all of these variables have provided functions to retrieve directly, often only via the generated stresses and strains which may not be the same as the value specified to actually generate that profile (i.e. in the case of confined concrete models as noted in #47 i.e. `max(self.strains) != self.ultimate_strain`. 

Will resolve on this basis @robbievanleeuwen.

Reply:
Addressed with dbe6e350751d300996170ca0703ec34de71617e3